src/fparser/two/utils.py

```diff
@@ -874,49 +874,128 @@ def match(op_pattern, rhs_cls, string, exclude_op_pattern=None):
 
 
 class BinaryOpBase(Base):
```

Comment:
Please update the copyright date on this file (and the other one if I forgot to say that).

Reply:
Updated.